tests/h/services/nipsa_test.py

```diff
@@ -51,6 +52,7 @@ def test_unflag_sets_nipsa_false(self, db_session, users):
 
         svc.unflag(users['renata'])
 
+        assert not svc.is_flagged('acct:renata@example.com')
```

Comment:
The way these tests are implemented don't necessarily guarantee that the internal cache got updated. They only black box verify that the user according to the service is flagged or unflagged as expected. For example you could remove the cache logic route from the is_flagged and remove the update cache logic from unflag and this test would still pass despite the cache not getting updated with the unflagged user. The alternative is a bit unpleasant tho as you would have to assert on a private variable. Thoughts?